src/Inventory/Asset/MainAsset.php

```diff
@@ -353,11 +353,18 @@ protected function prepareForBios($val)
         }
 
         $models_id = $this->getModelsFieldName();
-        if (property_exists($bios, 'smodel') && $bios->smodel != '') {
-            $val->$models_id = $bios->smodel;
-        } else if (property_exists($bios, 'mmodel') && $bios->mmodel != '') {
-            $val->$models_id = $bios->mmodel;
-            $val->model = $bios->mmodel;
+
+        $smodel = property_exists($bios, 'smodel') ? $bios->smodel : '';
+        $mmodel = property_exists($bios, 'mmodel') ? $bios->mmodel : '';
+        $model = array_unique(array_filter([$smodel, $mmodel]));
+        if ($model) {
+            if (count($model) == 1) {
+                $model = implode('', $model);
+            } else {
+                $model = sprintf("%s (%s)", ...$model);
+            }
```

Comment:
I think it would be easier to read, and so easier to maintain.

```suggestion
        $model = null;
        if (!empty($smodel) && empty($mmodel)) {
            $model = $smodel;
        } elseif (empty($smodel) && !empty($nmodel)) {
            $model = $nmodel;
        } elseif (!empty($smodel) && !empty($nmodel)) {
            $model = sprintf("%s (%s)", $smodel, $nmodel);
        }
        if (!empty($model)) {
```

Reply:
i rewrite some typos and variant if smodel == mmodel (its usually happens)